conda-store-server/conda_store_server/app.py

```diff
@@ -446,9 +446,13 @@ def register_environment(
             environment = orm.Environment(
                 name=specification.name,
                 namespace_id=namespace.id,
+                description=specification.spec["description"],
```

Comment:
Does this assume that a specification has to have a "description" field? Otherwise this PR looks great to me and I'm ready to merge this. I do want to make sure though that the `description` field is optional within a spec that is submitted.

Reply:
I confirm that the `description` field is optional.

I just made the test : 
- When submitting a yaml spec without the description, the env builds. 
- The rendering of the yaml **and** the content in the DB is : empty string 

### Rendering of the current UI : 

> ![Capture d’écran 2022-08-04 à 08 23 30](https://user-images.githubusercontent.com/756464/182777550-1dade9ff-49ce-4c26-9504-6b593f99c91e.png)


### Data in DB : 

> <img width="817" alt="Capture d’écran 2022-08-04 à 08 28 23" src="https://user-images.githubusercontent.com/756464/182778348-e84f1cfb-dc3d-4aab-afa2-6a7f84773153.png">

Reply:
Awesome thanks!